api/README.md

```diff
@@ -3,3 +3,10 @@
 ## JSON format
 
 JSON fields should be snake_case.
+
+## Documentation
+
+General guidelines, though not applicable in every case should be:
+
+- Request/Response model naming should follow: _[Subject][SubjectAction][Request|Response]_; I.E. **SessionListRequest, SessionListResponse, PasswordChangeRequest, PasswordResetResponse**...  
```

Comment:
What about `SessionListRequest` suffixing with `..DTO`
- all our entities has style _[Subject]DTO_ e.g. `IdentityDTO`, `ServiceInfoDTO`
- request/response naming differs: `IdentityListResponse` or `IdentityDTO` during cases them entity is response itself

Reply:
I am curious why you postfix DTO to some objects and not others?

I think if endpoint return type is `IntentityDTO[]` (list) to leave it like that for now. We can follow up on it later.

Reply:
DTO is just noise, doesn't add any value

Reply:
We had a face2face discussion with @Waldz about it. I came to understand that DTO suffixes have a scoping implications.  I.E. DTO objects don't leak internal information when passed around between "services" (or golang equivalent). 

Also there is always argument that certain prefix/suffix noise words add better discoverability when searching larger code bases at a cost of - noise.

I personally would be inclined to reduce noise and use strong names... as there is no real world equivalent of IdentityDTO as we tend to refer to it in day to day and business as simply `Identity`. The more code reflects actual business the better maintainability it has and it is easier to covert business ideas to features and vice versa.

That said I think this should another devbook PR discussion since we are expanding scope of this PR.